content/departments/people-talent/people-ops/process/how-we-engage-talent-outside-the-us/deel.md

```diff
@@ -0,0 +1,59 @@
+# Deel - PEO/EOR employment
+
+[Deel](https://www.deel.com/) is a privately-held payroll and compliance provider. That means anyone, anywhere can get hired and paid with Deel. Using a tech-enabled self-serve process, companies can legally hire full-time employees or independent contractors across the world. We use Deel to hire international employees and contractors. We refer to Deel and similar providers as Professional Employer Organizations (PEOs) or Employer of Record (EORs).
+
+Deel acts as the intermediary between you (the employee) and the company you're working for. Deel engages with both parties using separate agreements. While you perform your duties at the company, you get hired and paid by Deel.
+
+## How to get employed via a PEO/EOR?
+
+During your interview process, your recruiter will explain the different types of employment available. Should you choose to come in on a PEO/EOR, they will ask you for certain details and then notify you as soon as the request has been approved.
+
+After the People Ops team and the Finance team set everything up for you on the backend, Deel will contact you, notify you and request your signature. You can read about the onboarding task [here](https://help.letsdeel.com/hc/en-gb/articles/4414311737745-What-will-my-onboarding-process-look-like-).
+
+### Point of contact
+
+The recruiter will let you know that Deel will contact you.
+
+You will be assigned a Deel Onboarding Specialist. They will ask you to create an profile in Deel and send over compliance documentation. All of this must be done before your start date, you won’t be able to start until this is done. Read about [how to sign up as an employee](https://help.letsdeel.com/hc/en-gb/articles/5779651597585-How-to-sign-up-as-an-Employee) here.
+
+If there are any delays, Deel will notify the People Ops team and propose a new start date. Your point of contact for any questions regarding employment will be answered by the Onboarding Specialist.
+
+## FAQs
+
+#### Why is there no mention of equity or stocks in my contract?
+
+Since your employment contract is with Deel, any equity grants or stock options offered by your company will need a separate agreement between you and them.
```

Comment:
```suggestion
Since your employment contract is with Deel, any equity grants or stock options offered by your company will need a separate agreement between you and your company.
```